src/HasTranslations.php

```diff
@@ -91,6 +91,18 @@ public function getTranslations($key) : array
         return json_decode($this->getAttributes()[$key] ?? '' ?: '{}', true) ?: [];
     }
 
+    public function getAllTranslations() : array
+    {
+        $translations = [];
+
+        foreach ($this->getTranslatableAttributes() as $attribute) {
```

Comment:
Could you replace the `foreach` loop with a map operation?

Reply:
Yes, great suggestion. Thank you.